### DISCUSSION TITLE
Stream Dolby Atmos on PC

### DISCUSSION THREAD

**Discussion by cosmicvarion** [tidal-music/discussions#81] [Q&A]
Is there a way to Stream max of Dolby Atmos on PC (Windows 11)? I have Dolby Access installed and a Dolby Atmos for headphones subscription, in Settings > System > Sound > Properties, my headphones' Spatial sound Type is set to Dolby Atmos for Headphones, and I can stream Dolby Atmos audio with the examples in Dolby Access. 

However, though I can stream MQA, etc., I do not Stream Dolby Atmos in the Tidal app (installed from Microsoft Store) even from say the Top 50: Dolby Atmos playlist (or there's no indicator I'm streaming Dolby Atmos as there is on iOS). Is it not available?

**Comment by stefanmoro** [comment]
Hey @cosmicvarion,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
